composer.lock

```diff
@@ -1763,5 +1815,8 @@
     "platform": {
         "php": "^7.1"
     },
-    "platform-dev": []
+    "platform-dev": [],
+    "platform-overrides": {
+        "php": "7.1.0"
```

Comment:
did you need help with php 71?
overrides is not so good for the testing on different versions

Reply:
I agree. Locally I have PHP 7.4 and forced on composer global PHP 7.1.0.

If you can help I will appreciate. Which is the best approach to generate `composer.lock` without "writting" `platform.php` on it? Should be removed mannually this entry?